### PR TITLE
Added full test coverage for cli.py and iam.py

### PR DESCRIPTION
**Pull request by kelseylam**
Wrote full coverage unittests for cli.py and iam.py and added setup.cfg.

### CODE REVIEW

File: test/iam_test.py [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+        Checks if get groups, delete_user_from group, get_keys, delete_keys, and delete_user are called.
+        """
+        self.iam.delete_user(self.TEST_USER)
+
+        mock_get_groups.assert_called_once_with(self.TEST_USER)
+        groups = mock_get_groups.return_value
+        calls = []
+        for group in groups:
+            calls.append(call(self.TEST_USER, group['GroupName']))
+        mock_delete_from_group.assert_has_calls(calls)
```
Comment (hanpeter) [inline test/iam_test.py:251]: This will fail if remove the user from other groups as well.

File: test/iam_test.py [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+    @patch('krux_iam.iam.get_group')
+    @patch('krux_iam.iam.add_boto_cli_arguments')
+    def test_get_cli_arguments_no_boto(self, mock_add_boto, mock_get_group):
+        """
+        Test add_iam_cli_arguments with include_boto_arguments = False
+        """
+        parser = MagicMock()
+
+        add_iam_cli_arguments(parser, False)
```
Comment (hanpeter) [inline test/iam_test.py:185]: Try passing `True` here. Does your test fail?

**Comment by dcrampton-sfdc** [comment]
LGTM, but you'll want more eyeballs on it.

File: test/iam_test.py (hunk 2 above)
Comment (kelseylam) [inline test/iam_test.py:185]: It fails.

Comment (hanpeter) [inline test/iam_test.py:185]: Sorry, misread your code. My fault.

**Comment by dcrampton-sfdc** [comment]
Still LGTM, but you'll sill want to have more eyeballs than mine on it. :)

**Comment by hanpeter** [comment]
:+1:

**Pull request merged**